compositor_common/src/error.rs

```diff
@@ -64,6 +64,12 @@ pub enum BuiltinSpecValidationError {
     FixedLayoutLeftRightOnlyOne,
     #[error("Nodes that use transformation \"transform_to_resolution\" need to have exactly one input pad.")]
     TransformToResolutionExactlyOneInput,
+    #[error(
+        "Nodes that use transformation \"tiled_layout\" need to have at most {TILED_LAYOUT_MAX_INPUTS_COUNT}."
```

Comment:
"Transformation \"tiled_layout\" supports up to {TILED_LAYOUT_MAX_INPUTS_COUNT} inputs."